src/components/course/course-header/tests/DeprecatedCourseRunCard.test.jsx

```diff
@@ -45,16 +44,6 @@ jest.mock('../../../app/data', () => ({
 }));
 
 const INITIAL_APP_STATE = initialAppState({});
-const defaultCourse = mockCourseState({});
```

Comment:
[inform] For now, simply removing the unused function args/vars to get linting to pass; the tests still fail though. @brobro10000 is tackling the fixing of the actual tests. This might result in a minor merge conflict, but should be straightforward to resolve.